src/database/seeders/20200206095927-dummy-user.js

```diff
@@ -1,8 +1,11 @@
 export default {
   up: (queryInterface) => queryInterface.bulkInsert('Users', [{
-    firstName: 'Octapus',
-    lastName: 'Octapus',
-    email: 'octapus@andela.com',
+    userID: 'd0a051d9-447a-49a8-aebc-7e1b031afd62',
+    firstName: 'Octopus',
+    lastName: 'Octopus',
+    email: 'octopus@andela.com',
+    password: '123',
+    isVerified: false,
     createdAt: new Date(),
     updatedAt: new Date()
```

Comment:
is it possible that you may add a `method` too?

Reply:
if possible too add a hashed password 
for 'password: '123'' won't have any use

Reply:
@blaise82 Thank you, it is done